src/content/docs/accounts/accounts-billing/account-structure/add-accounts.mdx

```diff
@@ -0,0 +1,13 @@
+---
+title: Add accounts
+translate:
+  - jp
+metaDescription: How to add accounts to your New Relic organization. 
+---
+
+How you add accounts to your New Relic organization depends on the type of [account/user model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model/#user-models) your users are on: 
+
+* New Relic One user model: if you're on our newer user model, you currently can't add accounts using the UI but must talk to your account representative. For more information, see [Organization structure](/docs/accounts/accounts-billing/account-structure/new-relic-account-structure). 
```

Comment:
Feels like you might want to add some info about how someone might contact their account representative?

Reply:
we actually don't put that in docs as the thinking is: Pro and Enterprise account people know how to contact their assigned reps, and if they don't, they'd just need to find out, cause they'd always have a rep. the ability to add acconts is something only those higher tiers can do.